core/tests/protractor/accessibility.js

```diff
@@ -41,6 +67,312 @@ describe('screenreader and keyboard user accessibility features', function() {
         .getAttribute('id'));
   });
 
+  it('should test navigation shortcuts for the community-library page',
+    async function() {
+      await browser.get('community-library');
+      await waitForUrlRedirection('community-library');
+
+      await triggerKeys('0');
+      await waitForUrlRedirection(getStartedUrl);
+      expect(await browser.getCurrentUrl()).toEqual(getStartedUrl);
```

Comment:
Can you wait for the URL to match exactly (and not just contain) the expected URL? Then the `expect(...` would be unnecessary

Reply:
done and removed the unnecessary expects